uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/API/Wrapper/LeaveRequestDatesTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+use CRM_HRLeaveAndAbsences_API_Wrapper_LeaveRequestDates as LeaveRequestDates;
+
+/**
+ * Class CRM_HRLeaveAndAbsences_API_Wrapper_LeaveRequestDates
```

Comment:
Is this line intentional?

Reply:
It was added automatically by PhpStorm and I didn't bother to remove it. Removing it now